### DISCUSSION TITLE
set disk quotas for each shared directory

### DISCUSSION THREAD

**Discussion by hnagasawa** [Netatalk/netatalk#1062] [Q&A] [answered]
Is there an option to set disk quotas for each shared directory?
The manual says that vol size limit option is for Time Machine.

**Comment by rdmark** [comment, marked as answer]
The “vol size limit” option is not a true quota feature. It just limits the amount of storage space that is reported to the client. 

Netatalk 3.1 has theoretical support for sunrpc quota, as well as libtirpc quota (as of 3.1.16). You would define quotas through those system libraries and Netatalk will theoretically obey them. To be honest, I don’t know if it works properly.

Netatalk will need to be compiled with `--enable-quota` and also `--with-libtirpc` for the latter library (mandatory on Debian and maybe other Linux flavors).

Please note that we decided to remove quota functionality altogether from the next major release onwards (3.2) because it had been broken for many years and no one complained…

**Comment by rdmark** [comment]
For the record: quota functionality has been restored in 3.x with https://github.com/Netatalk/netatalk/pull/1034 and will be part of the upcoming 3.2.0 release.
